cli/man/splinter-registry-add.1.md

```diff
@@ -0,0 +1,128 @@
+% SPLINTER-REGISTRY-ADD(1) Cargill, Incorporated | Splinter Commands
+<!--
+  Copyright 2018-2021 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+NAME
+====
+
+**splinter-registry-add** — Add a node to the local registry
+
+SYNOPSIS
+========
+
+**splinter registry add** \[**FLAGS**\] \[**OPTIONS**\] IDENTITY
+
+DESCRIPTION
+===========
+
+Add a new node to the local node registry. The node may be entirely new to the
+registry, or it may be copied from the remote registries with the
+`--from-remote` flag. If the `--from-remote` flag is used the `--display-name`,
+`--endpoint`, `--key` and `--metadata` options may not be used to alter the
+node being copied from the remote registry. When run, the command will
+display the resulting changes as confirmation.
+
+FLAGS
+=====
+`--dry-run`
+: Shows the expected changes without submitting the node.
+
+`--from-remote`
+: Copies an existing node definition from the remote registries. 
+
+`-h`, `--help`
+: Prints help information
+
+`-q`, `--quiet`
+: Decreases verbosity (the opposite of -v). When specified, only errors or
+  warnings will be output.
+
+`-V`, `--version`
+: Prints version information
+
+`-v`
+: Increases verbosity (the opposite of -q). Specify multiple times for more
+  output.
+
+OPTIONS
+=======
+
+`--display-name` DISPLAY_NAME
+: Sets a human-readable name for the new node. If not provided, a default value
+based on the node's ID will be used.
+
+`--endpoint ENDPOINT`
+: Adds a network endpoint for the new node. At least one endpoint must be
+provided, and all endpoints must be non-empty and unique in the registry (two
+nodes cannot share the same endpoint). Repeat this option to specify multiple endpoints.
+
+`--key-file KEY`
+: Add the public key to the new node. At least one key must be provided, and all
+keys must be non-empty. Repeat this option to specify multiple keys.
+
+`-k`, `--key KEY`
+: Name or path of private key to be used for REST API authorization.
+
+`--metadata METADATA_STRING`
+: Adds the metadata to the new node, using the format
+`METADATA_KEY:METADATA_VALUE`. If an entry for the given `METADATA_KEY` already exists, it will be replaced. Repeat this option to specify multiple metadata
```

Comment:
This line is too long, should be wrapped

Reply:
Fixed